src/main/java/com/mebigfatguy/fbcontrib/utils/SerialVersionCalc.java

```diff
@@ -40,71 +41,80 @@
         CLASS, METHOD, FIELD
     }
 
+    private static void writeInterfaces(DataOutput out, JavaClass cls) throws IOException {
+        String[] infs = cls.getInterfaceNames();
+        if (infs.length > 0) {
+            infs = infs.clone();
+            Arrays.sort(infs);
+            for (String inf : infs) {
+                out.writeUTF(inf);
+            }
+        }
+    }
+
+    private static void writeFields(DataOutput out, JavaClass cls) throws IOException {
+        Field[] fields = cls.getFields();
+        if (fields.length > 0) {
+            fields = fields.clone();
+            Arrays.sort(fields, new FieldSorter());
+            for (Field field : fields) {
+                if (!field.isPrivate() || (!field.isStatic() && !field.isTransient())) {
```

Comment:
So...this outputs all public/protected/package fields, plus private non-transient instance fields? I have to think there's a simpler way to express that.